packages/eas-cli/src/credentials/ios/actions/SetupBuildCredentials.ts

```diff
@@ -26,10 +26,13 @@ interface Options {
   skipCredentialsCheck?: boolean;
 }
 
-export class SetupBuildCredentials implements Action {
+export class SetupBuildCredentials implements Action<IosAppBuildCredentialsFragment> {
```

Comment:
SetupBuildCredentials.runAsync should return IosAppBuildCredentialsFragment but field shouldnt be optional